packages/pigment-css-react/tests/generateAtomics.test.js

```diff
@@ -3,210 +3,39 @@ import { atomics } from '../src/generateAtomics';
 
 const atomic = atomics({
   styles: {
-    display: {
-      flex: {
-        xs: 'display-flex-xs',
-        $$default: 'display-flex-xs',
-        sm: 'display-flex-sm',
-        md: 'display-flex-md',
-        lg: 'display-flex-lg',
-        xl: 'display-flex-xl',
-      },
-      'inline-flex': {
-        xs: 'display-inline-flex-xs',
-        $$default: 'display-inline-flex-xs',
-        sm: 'display-inline-flex-sm',
-        md: 'display-inline-flex-md',
-        lg: 'display-inline-flex-lg',
-        xl: 'display-inline-flex-xl',
-      },
-    },
     flexDirection: {
       column: {
         xs: 'flex-direction-column-xs',
-        $$default: 'flex-direction-column-xs',
```

Comment:
What are you assuming the default breakpoint in case someone passed this ?

```tsx
<Stack spacing={1} />
```

Reply:
Where is that `$$default` coming from? Isn't the `defaultCondition` enough to know the default breakpoint?

Reply:
Ok. It was a remnant of the first implementation of the `Box` component.